v5/schemas.go

```diff
@@ -610,6 +611,7 @@ type PartyCreate struct {
 	Subject               string          `json:"subject,omitempty"`
 	SuppressInviteEmail   *bool           `json:"suppress_invite_email,omitempty"`
 	TeamId                string          `json:"team_id,omitempty"`
+	ClientVisibility      int             `json:"client_visibility,omitempty"`
```

Comment:
Have you updated the external_api? I found the external_api schema has not been changed.
https://github.com/UrbanCompass/compass-glide-devapp/blob/development/webapp/web/external_api/transactions/transaction.py#L133-L152

Reply:
Hi Wei, no I haven't let me do it in my PR right now

Reply:
If we add a party with this param and do not modify the external-api, it will raise a 500 error.
![image](https://github.com/user-attachments/assets/15030935-0d92-435b-b6b0-4be2cce74020)
![image](https://github.com/user-attachments/assets/ddbfb650-12cb-4b27-b680-9e770aa6bc8a)

Reply:
Added it here in the last commit, maybe you could take a look at that party to make sure is ok?
 https://github.com/UrbanCompass/compass-glide-devapp/pull/1525

Reply:
I think you'd better notify the people who will be using this param